ocs_ci/ocs/bucket_utils.py

```diff
@@ -1424,3 +1424,66 @@ def write_random_test_objects_to_bucket(
         signed_request_creds=s3_creds,
     )
     return obj_lst
+
+
+def write_data_to_buckets_and_check_integrity(
+    mcg_obj, awscli_pod, bucket_factory, bucketclass_dict, number_of_buckets
+):
+    """
+    Write data on several buckets and check data integrity with md5sum
+
+    Returns:
+        dictionary: Data on buckets including downloaded_files, original_dir,
+        result_dir, awscli_pod
+
+    """
+    buckets_data_dic = dict()
+    for bucket_id in range(1, number_of_buckets + 1):
+        bucketname = bucket_factory(1, bucketclass=bucketclass_dict)[0].name
+        original_dir = f"/original{bucket_id}"
+        result_dir = f"/result{bucket_id}"
+        awscli_pod.exec_cmd_on_pod(command=f"mkdir {result_dir}")
+        # Retrieve a list of all objects on the test-objects bucket and
+        # downloads them to the pod
+        full_object_path = f"s3://{bucketname}"
+        downloaded_files = retrieve_test_objects_to_pod(awscli_pod, original_dir)
```

Comment:
`retrieve_test_objects_to_pod()` is no longer in use.
Thje objects are now integrated into the S3 CLI pod image, please replace with the `test_directory_setup` fixture (in the test's scope)